### ISSUE TITLE
open detected links with ctrl+click in addition to/instead of double click

### ISSUE THREAD

**Issue by apfelchips** [notepad-plus-plus/notepad-plus-plus#9993]
### Description of the Issue
Most terminals (eg. Windows Terminal, ConEmu) and other text editors (eg. VSCode) that can detect urls use use a ctrl+click to open them. Currently NP++ only supports double clicking.

Please add this functionality as an additional default and/or provide the user the option:
- link detection off
- link detection on: double click to open
- link detection on: ctrl+click to open _(current default)_
- link detection on: double click to open + double ctrl+click to open _(proposed default)_

This change shouldn't interfere with the current UX, not disturbing current user expectations.
It could possibly interfere with the ctrl+drag multi-line cursor, but that feature is off by default and the same can already be accomplished by using alt+drag.

Thanks. I love the lightness of NPP but this litte UX thing really irks me. 

### Steps to Reproduce the Issue

1. hover over underlined url 
2. ctrl+click

### Expected Behavior

open link in default browser like with double click

### Actual Behavior

noting

### Debug Information
Notepad++ v7.9.5   (64-bit)
Build time : Mar 21 2021 - 02:13:17
Path : C:\Program Files\Notepad++\notepad++.exe
Admin mode : ON
Local Conf mode : OFF
OS Name : Windows 10 Pro (64-bit) 
OS Version : 2009
OS Build : 19043.1052
Current ANSI codepage : 1252
Plugins : HexEditor.dll mimeTools.dll NppConverter.dll NppEditorConfig.dll NppExport.dll NPPJSONViewer.dll NppMarkdownPanel.dll OpenSelection.dll XMLTools.dll

**Comment by ArkadiuszMichalski** [comment]
I agree with this, I often need URL detection, but also I have to copy some part of it. Due to this double-clicking, I often open the link instead of selecting part of it. This is annoying because we can't turn on / off URL detection quickly (need jump to Preferences window).

For me the solution is:
- add option to open URL with middle click (Ctrl+click may conflict with multiediting so I don't recommend it)
- expose Enable/Dissable URL detection in a more accessible place (somewhere in Menu), also with an possibility to set a shortcut.

**Comment by Shayan-To** [comment]
This is the single feature request I have from Notepad++. I nearly always use double-click for selection (if you double-click and drag, your selection will snap to word boundaries), and it is very very annoying around links.

**Comment by alankilborn** [comment]
@Shayan-To said:

>(if you double-click and drag, your selection will snap to word boundaries)

I don't understand what this means, but I'd like to.

**Comment by darkzbaron** [comment]
That would be great to be able to customize this
